api/app/Models/Pool.php

```diff
@@ -189,4 +181,13 @@ public function scopeAuthorizedToView(Builder $query)
 
         return $query;
     }
+
+    public function scopeNotArchived(Builder $query, ?array $args)
```

Comment:
You can leave out the $args prop since it's not being used.

Reply:
:+1: 
[Remove unused parameter](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7399/commits/97984ab44ddf354bf092c781082d2017b2622bbb)